docs/1.concepts/account-abstraction/chainsig.md

```diff
@@ -0,0 +1,114 @@
+---
+id: chain-signatures
+title: Chain Signatures
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+Chain Signatures are a method to allow a smart contract to call an async method. For example:
+
+```rust
+async fn sign(
+    payload: Bytes,
+    key_derivation: Bytes,
+    key_type: ECDSA) -> Signature
+```
+
+The signature is calculated on the data payload using a user-provided key. The signature output is always the same, as long as the caller and payload remains unchanged.
+
+:::info
+The signature is an ECDSA signature on the Secp256k1 curve.
+:::
+
+## Use cases
```

Comment:
One of my concerns here is that while these use cases are still valid, they're fairly convuluted and worse they're pretty remote from how you should actually build these smart contracts.

Reply:
We can change /replace these use cases with others that fit better. We can share some ideas during our next call.

Reply:
Hi @DavidM-D , following up on this draft, do you want to replace any of the current use-cases, or want to add some new ones?

Also, let me know if you want to add more information to this article. (please link reference documents so I don't end up using outdated info)

Reply:
I think adding in key markets is a good idea, so the ability to change the ownership of Bitcoin/Ethereum accounts to trade assets across chains without doing on chain transactions

Reply:
Hi @DavidM-D , I have added a section about trading assets without transactions. 

Preview link:
https://docs-preview-pr-1704.onrender.com/concepts/chain-abstraction/signatures#trade-blockchain-assets-without-transactions